tests/testTrotErr.py

```diff
@@ -0,0 +1,202 @@
+'''
```

Comment:
Testing will not comply with the automatic testing suite here. 
It needs to have the following structure:

Every function in the file, that is names "test_xyz" where xyz can be anything will be executed in the testing suite. In this function you basically do the following:

1. Compute something
2. Test if the result is correct with an assert statement.

Here is an example:

def test_addition():
    test = 1 + 2
    assert test == 3

Ideally tests work from scratch, i.e. they compute everything themselves, so that we can test for global consistency in the package.
So in your case you would need something like:

test_trotter_errors():
    molecule = tq.Molecule(....)
    # do what you are doing in your example notebook
    ....
   # took the number from your notebook
    assert numpy.isclose(result_computer_here, 0.210588476484733, atol=1.e-4)
    
hope that makes sense
Otherwise let me know and I can explain more.

The content of the file here is (as far as I see) reproducing plots form your paper. You could add this to your directly in the tutorials and add some remarks in the readme file (I would then also upload the data there). 
Here in the testing suite we can't use it (because the suite is not able to check if the plots are correct).

Reply:
Thanks a lot for the feedback! I have changed the testing file accordingly. Also, I changed the tutorial notebook and added the auxiliary files in the same directory where the tutorial is